webapp/src/lib/encoding.test.ts

```diff
@@ -0,0 +1,72 @@
+import { describe, expect, it } from "vitest";
+import {
+	bytes32ToAddress,
+	getChecksummedAddress,
+	getShortAddress,
+	compactSignatureToFullSignature,
+} from "./encoding";
+import { ethers } from "ethers";
+import type { HarbourSignature } from "./types";
+
+describe("encoding", () => {
+	const lower = "0xdeadbeefdeadbeefdeadbeefdeadbeefdeadbeef";
```

Comment:
nit: UPPER CASE constants. Not sure if this would be handled by our formatter, try running `npm run check`